### ISSUE TITLE
Cannot pip install torchvision==0.8.1+cu110 (python 3.9)

### ISSUE THREAD

**Issue by ihmc3jn09hk** [pytorch/vision#3046]
## 🐛 Bug

Install Pytorch stable failed 

```console
$ pip3 install torch==1.7.0+cu110 torchvision==0.8.1+cu110 torchaudio===0.7.0 -f https://download.pytorch.org/whl/torch_stable.html
Looking in links: https://download.pytorch.org/whl/torch_stable.html
ERROR: Could not find a version that satisfies the requirement torch==1.7.0+cu110 (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2)
ERROR: No matching distribution found for torch==1.7.0+cu110

```
## To Reproduce

Use the following command
```console
1. pip3 install torch==1.7.0+cu110 torchvision==0.8.1+cu110 torchaudio===0.7.0 -f https://download.pytorch.org/whl/torch_stable.html
```
After browsing through https://download.pytorch.org/whl/torch_stable.html, I wonder the issue is related to Python 3.9. 
The tested environment is:
Python : 3.9 
Cuda 11.1
Ubuntu 20.04.

**Comment by vfdev-5** [comment]
@ihmc3jn09hk yes, it is related to Python 3.9. The binaries for Pytorch, Torchvision and others are only available for Python 3.6, 3.7 and 3.8.

**Comment by ihmc3jn09hk** [comment]
> @ihmc3jn09hk yes, it is related to Python 3.9. The binaries for Pytorch, Torchvision and others are only available for Python 3.6, 3.7 and 3.8.

Thank you. Will there be a release day of support for Python 3.9?

**Comment by datumbox** [comment]
Thanks for reporting @ihmc3jn09hk. 

Yes we are looking into it. There is already a PR #3041 that adds Python 3.9 on the CI. I don't have a concrete date yet, but it's something we will support on the future. You can be notified if you subscribe to the aforementioned PR.
